apps/web/src/App.tsx

```diff
@@ -180,12 +193,142 @@ function App() {
+                              <QuickStartHeader>
+                                <GeneralStarter />
+                              </QuickStartHeader>
+                            </SpotLight>
+                          </RequiredAuth>
+                        }
+                      />
                       <Route
                         path="/quickstart"
                         element={
                           <RequiredAuth>
                             <SpotLight>
-                              <QuickStartPage />
+                              <QuickStartHeader
+                                title={welcomeDescription}
+                                description={'What would you like to build?'}
+                              >
+                                <QuickStartCards
+                                  cells={[
+                                    {
+                                      navIcon: BellGradient,
+                                      description: 'Notification Center',
+                                      navigateTo: '/quickstart/notification-center',
+                                    },
+                                    {
+                                      navIcon: NestedSquares,
+                                      description: 'Other',
+                                      navigateTo: '/general-started',
+                                    },
+                                  ]}
+                                />
+                              </QuickStartHeader>
+                            </SpotLight>
+                          </RequiredAuth>
+                        }
+                      />
+                      <Route
+                        path="/quickstart/notification-center/set-up"
+                        element={
+                          <RequiredAuth>
+                            <SpotLight>
+                              <QuickStartHeader title={welcomeDescription} description={<ImplementationDescription />}>
+                                <QuickStartCards
+                                  cells={[
+                                    {
+                                      imagePath: `/static/images/frameworks/dark/react.png`,
+                                      navigateTo: '/quickstart/notification-center/set-up/react',
+                                    },
+                                    {
+                                      imagePath: `/static/images/frameworks/dark/angular.png`,
+                                      navigateTo: '/quickstart/notification-center/set-up/angular',
+                                    },
+                                    {
+                                      imagePath: `/static/images/frameworks/dark/vue.png`,
+                                      navigateTo: '/quickstart/notification-center/set-up/vue',
+                                    },
+                                  ]}
```

Comment:
Maybe it will be a good idea to refactor this one, move the `cells` param to consts file or create a component that will contain this logic (+ moving cells to consts)

Reply:
same for other routes.